.github/workflows/build-ros.yml

```
@@ -1,65 +1,66 @@
-          - docker_image: ubuntu:noble
-            ros_distribution: rolling
-            use_ros_testing: true
-
     container:
       image: ${{ matrix.docker_image }}
 
     steps:
       - name: Checkout
+        env:
+          TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:3:19: Double quote to prevent globbing and word splitting

(shellcheck)